lib/davinci_pdex_test_kit/pdex_payer_server_suite.rb

```diff
@@ -38,14 +38,15 @@ class PDexPayerServerSuite < Inferno::TestSuite
     input :url,
           title: 'FHIR Server Base Url'
 
-    input :credentials,
+    input :smart_auth_info,
           title: 'OAuth Credentials',
-          type: :oauth_credentials,
-          optional: true
+          type: :auth_info,
+          optional: true,
+          default: {auth_type: 'backend_services'}.to_json
```

Comment:
We don't have to do that anymore and we shouldn't because it causes problems.
```
options: {
  components: [
    {
      name: :auth_type,
      default: 'backend_services'
    }
  ]
}